ctapipe/image/timing.py

```diff
@@ -76,12 +76,13 @@ def sigma_clipping_linreg(x, y, kappa=3, n_iter=3):
     """
 
     a, b, cov = linear_regression(x, y)
+    mask = np.ones(len(x), dtype=np.bool_)
```

Comment:
can use `np.ones_like(x, dtype=np.bool_)` here

Reply:
ones_like was a bit slower and we only support 1d here anyways.